.github/ISSUE_TEMPLATE/issue_tracker_template.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Onboarding Tracker⛴️
+about: To be created by API Backlog codeowners after a new repository got approved
+title: '[Onboarding Tracker] $repo name$'
+labels: 'API Onboarding Tracker'
+assignees: ''
+
+---
+
+<!-- API & repository Tracker Issue -->
+**API information**
+- **Name**: XXXX
+- **Mail List:** XXXX
+- **Repository**: XXXX
+- **Wiki Page**: XXXX
+- **API owner**: TBD
+- **API proposal**: https://github.com/camaraproject/APIBacklog/blob/main/documentation/XXXX.md
+- **Original Issue**: https://github.com/camaraproject/APIBacklog/issues/XXXX
+
+**Action List**
+- [ ] Get nominations of initial code owner(s) (one mandatory) (API owner)
+- [ ] Update of APIBacklog.md with new status ("Onboarding")
+- [ ] Create wiki page and mailing list (send mail to support@camaraproject.org)
```

Comment:
@wrathwolf @eharrison24 

Should we:
a) split this line into two different action for wiki page and mailing list
b) is it enough that you got notified via GitHub about a new Onboarding Tracker (via cc: @camaraproject/admins) or do you need a separate mail to support@camaraproject.org about the issue?

Reply:
@albertoramosmonagas Good point ... I proposed a new suggestion to split it even into three lines ... one for the request of wiki and mailing list and two if they are created. If ok for you we can proceed (I wasn't aware that this PR is still dangling around).